app/pages/project/index.cjsx

```diff
@@ -173,9 +170,15 @@ ProjectPageController = React.createClass
 
   getSelectedWorkflow: (project, preferences) ->
     @setState({ loadingSelectedWorkflow: true })
-    # preference workflow query, then user selected workflow, then project owner set workflow, then default workflow
+    # preference workflow query by admin/owner/collab, then workflow query if in project settings, then user selected workflow, then project owner set workflow, then default workflow
     # if none of those are set, select random workflow
-    if @props.location.query?.workflow? and @canFetchWorkflowByQuery(project, @props.user)
+    activeFilter = true
+    if @props.location.query?.workflow? and @checkUserRoles(project, @props.user)
+      selectedWorkflowID = @props.location.query.workflow
+      activeFilter = false
+      unless preferences?.preferences.selected_workflow is selectedWorkflowID
```

Comment:
I think we still want this behavior even if admin/owner/collab because that user might just be wanting to classify on their own project and storing the workflow id in the prefs means it'll load up the correctly selected workflow if they browse between talk and back, etc. If they revisit w/o the query param for the workflow, the behavior will clear out the workflow from their prefs if inactive.